### DISCUSSION TITLE
Why Grafana rollout operator is required for mimir?

### DISCUSSION THREAD

**Discussion by rosh0450** [grafana/mimir#7728] [Help and support] [answered]
Can't mimir be installed without rollout operator?
is it recommended to do this??

**Comment by dimitarvdimitrov** [comment, marked as answer]
the rollout operator is needed only for zone-aware replication. This paragraph explains it better https://grafana.com/docs/mimir/latest/configure/configure-zone-aware-replication/#kubernetes-operator-for-simplifying-rollouts-of-zone-aware-components
